force-app/test/jest-mocks/lightning/alert.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
```

Comment:
Drop copyright header on the 4 mock files.

Reply:
Changes Done!